lib/Xud.ts

```diff
@@ -38,6 +39,9 @@ class Xud {
     await this.config.load();
     this.logger.info('config loaded');
 
+    /** Loads environment variables from the file .env */
+    dotenv.config();
+
```

Comment:
Please move this to the top of the file, after the imports and before the class definition. it's should be called once per process.